### PR TITLE
Tkt 00003 - Header Changes with navigation, ribbons, buttons

### PR DESCRIPTION
**Pull request by ishanivardhan**
@cehwitham - As discussed earlier, I'm pushing what's done of Header for review.
Button styles & alignment overall is not perfect but will be once Harry completes his work.

### CODE REVIEW

**Comment by ishanivardhan** [comment]
@cehwitham - any update on this?

File: style/middleware/_middleware.header.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
+ ** TO-DO: Few bits with this comment to be removed once implemented in Core Nightingale
```
Comment (cehwitham) [inline style/middleware/_middleware.header.scss:2]: Have we got issues raised in the repo to revisit these items?

File: templates/header.mustache [hunk 2]
```diff
@@ -17,44 +17,68 @@
...
+                        <nav class="c-nav-primary__sub  jsNavSub" id="item-2-sub" role="group" aria-label="sub navigation menu">
+
+                            <div class="o-wrapper">
+
+                                <div class="o-layout">
+                                    <div class="o-layout__item  u-6/12@lg">
+
+                                                <ul class="c-nav-primary__sub-links">
+                                                    <li>
+                                                        {{{availablecourses}}}
```
Comment (cehwitham) [inline templates/header.mustache:53]: If there are no available courses does this leave us with empty markup or does it return a "no courses" message?

File: style/middleware/_middleware.header.scss (hunk 1 above)
Comment (ishanivardhan) [reply style/middleware/_middleware.header.scss:2]: Yes @cehwitham [tkt-00024](https://github.com/NHSLeadership/moodle-theme_nightingale/issues/24)

File: templates/header.mustache (hunk 2 above)
Comment (ishanivardhan) [reply templates/header.mustache:53]: @cehwitham  No courses check has been added. Pls review the latest commit below.

**Review by cehwitham — APPROVED** [review]

**Pull request merged**